features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,21 @@
+Given("I visit the landing page") do
+  visit root_path
+end
+
+Given("I click {string}") do |element|
+  click_on element
+end
+
+Given("I visit the menu page") do
+  visit products_path
```

Comment:
whats the difference between the root_path and this path?

Reply:
root_path is to the landing page, products_path is to the menu page :)